enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -2194,6 +2187,23 @@ func (c *FirecrackerContainer) Exec(ctx context.Context, cmd *repb.Command, stdi
 	return result
 }
 
+func (c *FirecrackerContainer) initialize(ctx context.Context, result *interfaces.CommandResult) error {
```

Comment:
nit: Could we call this something that better highlights its purpose? Like `resetGuestState`